auto.def

```diff
@@ -113,6 +110,13 @@ options {
   pkgconf=0                 => "Use pkg-config during configure"
 # Enable all options
   everything=0              => "Enable all options"
+# Debug options
+  debug-backtrace=0         => "DEBUG: Enable backtrace support with libunwind"
+  with-backtrace:path       => "Location of libunwind"
+  debug-graphviz=0          => "DEBUG: Enable Graphviz dump"
+  debug-notify=0            => "DEBUG: Enable Notifications dump"
+  debug-parse-test=0        => "DEBUG: Enable 'neomutt -T' for config testing"
+  debug-window=0            => "DEBUG: Enable Windows dump"
```

Comment:
I would spell `windows` without the upper case `W`, to avoid triggering unwanted attentions ;)

Reply:
Done